### DISCUSSION TITLE
Formatting the code

### DISCUSSION THREAD

**Discussion by mustafa9514** [pawelsalawa/sqlitestudio#4392] [General]
Formatting the code:

Is it possible to format the code in one line?

splitting into many lines gets very tedious and the code becomes unreadable?

**Comment by pawelsalawa** [comment]
Go to configuration dialog, go to Plugins and select "SQL Simple" to load the plugin:
![image](https://user-images.githubusercontent.com/481846/151993291-5549cdce-e052-4ec5-be18-2ef217c9134a.png)

Then go to Code Formatters and select this plugin:
![image](https://user-images.githubusercontent.com/481846/151993393-888c53f7-65cc-4f90-9810-efd528e3bbf0.png)

The confirm the configuration dialog.

**Reply by pawelsalawa** [reply]
Then also configure that plugin to disable new lines.
